{"uid":"f34f5e956cd798a1","name":"test_all_tasks_exist","fullName":"dags.tests.test_dag_mongo_fhir#test_all_tasks_exist","historyId":"b6a2849297fc9dd8d4580bccac46785e","time":{"start":1744376434994,"stop":1744376434996,"duration":2},"status":"broken","statusMessage":"AttributeError: 'NoneType' object has no attribute 'task_ids'","statusTrace":"dagbag = <airflow.models.dagbag.DagBag object at 0x0000022A7D612360>\n\n    def test_all_tasks_exist(dagbag):\n        dag = dagbag.get_dag('mongo_fhir_data_fetcher_multiple_dbs_sequential')\n    \n        # Resource and DB names\n        resources = [\n            'Patient', 'Encounter', 'ServiceRequest',\n            'Claim', 'Condition', 'Procedure',\n            'Observation', 'MedicationRequest'\n        ]\n        dbs = [\"FHIR_Resource\", \"KhonKaen_DB\", \"Chonburi_DB\"]\n    \n        for db_name in dbs:\n            for idx, resource in enumerate(resources):\n                task_id = f'fetch_{resource.lower()}_{db_name}_{idx+1}'\n>               assert task_id in dag.task_ids, f\"Task {task_id} not found in DAG.\"\nE               AttributeError: 'NoneType' object has no attribute 'task_ids'\n\ndags\\tests\\test_dag_mongo_fhir.py:30: AttributeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"dagbag","time":{"start":1744376432399,"stop":1744376434610,"duration":2211},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"parentSuite","value":"dags.tests"},{"name":"suite","value":"test_dag_mongo_fhir"},{"name":"host","value":"MSI"},{"name":"thread","value":"82684-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"dags.tests.test_dag_mongo_fhir"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"4ad2add79840b9c3","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'task_ids'","time":{"start":1744376287316,"stop":1744376287318,"duration":2}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"tags":[]},"source":"f34f5e956cd798a1.json","parameterValues":[]}